bin/glite-info-update-endpoints

```diff
@@ -89,13 +95,32 @@ def usage():
               -v --verbose Run in verbose mode
 
             """
+
+def get_url_data(url):
+    # python urllib2 introduced server certificate validation starting
+    # with version 2.7.9 and 3.4 (backported also e.g. to CentOS7). It
+    # is no longer possible to download HTTPS data without having server
+    # CA certificate in trusted store or explicitely disable verification.
+    if hasattr(ssl, 'create_default_context'):
+        capath = config.get('capath')
+        cafile = config.get('cafile')
+        if capath != None or cafile != None:
+            context = ssl.create_default_context(cafile=cafile, capath=capath)
+        else:
+            context = ssl._create_unverified_context()
+        return urllib2.urlopen(url, context=context).read()
+    else:
+        # older python versions doesn't really verify server certificate
+        return urllib2.urlopen(url).read()
+
 def get_osg_urls():
 
-    osg_goc_url = "http://myosg.grid.iu.edu/rgsummary/xml?datasource=summary&summary_attrs_showwlcg=on&all_resources=on&gridtype=on&gridtype_1=on&active=on&active_value=1&disable_value=1"
```

Comment:
I propose that constants like this be read from a config file or the environment. They shouldn't be defined in the source code.